src/platform/intel/cavs/platform.c

```diff
@@ -402,6 +402,7 @@ int platform_init(struct sof *sof)
 	shim_write(SHIM_LPSCTL, shim_read(SHIM_LPSCTL));
 
 #elif CONFIG_CANNONLAKE
+	clock_set_freq(CLK_CPU(cpu_get_id()), CLK_MAX_CPU_HZ);
```

Comment:
Can  you move it before ifdefs, so it will be for all platforms as it was before? (below `trace_point(TRACE_BOOT_PLATFORM_CPU_FREQ);`)